PowerPointLabs/PowerPointLabs/SyncLab/Views/SyncFormatPaneItem.xaml

```diff
@@ -7,15 +7,22 @@
              mc:Ignorable="d" 
              d:DesignHeight="60" d:DesignWidth="300"
              MouseDoubleClick="OnMouseDoubleClick">
-    <Grid x:Name="grid" ClipToBounds="True">
+    <Grid x:Name="grid" ClipToBounds="True" Background="Transparent">
+        <Grid.ToolTip>
+            <StackPanel>
+                <TextBlock x:Name="toolTipName" FontWeight="Bold" FontSize="14" Margin="0,0,0,5"/>
+                <Border BorderBrush="Silver" BorderThickness="0,1,0,0" Margin="0,8" />
+                <TextBlock x:Name="toolTipBody"/>
+            </StackPanel>
+        </Grid.ToolTip>
         <Grid.ColumnDefinitions>
             <ColumnDefinition x:Name="col1" Width="60" MaxWidth="60"/>
             <ColumnDefinition x:Name="col2" Width="*"/>
             <ColumnDefinition x:Name="col3" Width="100" MaxWidth="100"/>
         </Grid.ColumnDefinitions>
         <Image x:Name="imageBox" Grid.Column="0" Margin="10,10,0,10" OpacityMask="{DynamicResource {x:Static SystemColors.ControlBrushKey}}" HorizontalAlignment="Left" Width="40" Height="40"/>
         <Canvas x:Name="canvas" Grid.Column="1" >
-            <TextBlock x:Name="textBlock" Margin="0,20,0,20" MaxWidth="{Binding ActualWidth, ElementName=canvas}" HorizontalAlignment="Stretch" VerticalAlignment="Center" TextTrimming="CharacterEllipsis"/>
+            <TextBlock x:Name="textBlock" Margin="0,20,0,20" MaxWidth="{Binding ActualWidth, ElementName=canvas}" HorizontalAlignment="Stretch" VerticalAlignment="Center" IsHitTestVisible="False"/>
```

Comment:
Any reason why you removed `TextTrimming` here? If I enter a very very long name for the SyncLab entry, the title view on the pane no longer has the ellipses to signify that it's a long title, it just gets truncated.

Also, if I enter a very very long title, the tooltips box gets stretched all the way across the whole screen. While this doesn't affect any functionality, I think it will look nicer if we restricted the tooltips box to a max width that looks nice.

Screenshot of the stretched tooltips:
![tooltip-box-too-long](https://user-images.githubusercontent.com/39292809/51822409-2d230300-2317-11e9-892d-4c2fcb8a9a71.png)